### PR TITLE
Added 2-letter language code support to trimpairs.py

### PR DESCRIPTION
**Pull request by dolphingarlic**
This is for the GCI task https://codein.withgoogle.com/tasks/4736448721846272/?sp-organization=5288137455566848&sp-is_beginner=False

### CODE REVIEW

File: trimpairs.py [hunk 1]
```diff
@@ -3,72 +3,111 @@
...
+    FILTERED = filter_pairs("pairs.json.txt", LANG_DICT, ALT_CODES)
+    APERTIUM_FILE = open("apertiumPairs.json", "w")
 
-    apertiumFile.write("{\n")
-    apertiumFile.write('"type": "FeatureCollection",\n')
-    apertiumFile.write('"pairs": [\n')
-    for line in filtered[:-1]:
-        apertiumFile.write("\t"+line+"\n")
-        apertiumFile.write(",\n")
+    APERTIUM_FILE.write("{\n")
```
Comment (jonorthwash) [inline trimpairs.py]: Why did you rename these variables?

File: apertium-languages.tsv [hunk 2]
```diff
@@ -24,6 +24,7 @@ cos,42,9
 crh,45.3,34.4
 cym,52.5,-4
 dan,55.5,11.5
+deu,52,10
```
Comment (JPJPJPOPOP) [inline apertium-languages.tsv]: Just curious, where did you get this coordinate from? It looks right on the globe.

File: trimpairs.py [hunk 3]
```diff
@@ -3,72 +3,111 @@
...
         if lang1 in languages and lang2 in languages:
-            #getting rid of last comma
+            # Getting rid of last comma
             line[-1] = line[-1][:-1]
             joined = " ".join(line)
             filtered.append(joined)
 
-    pairFile.close()
+        elif (
+                lang1 in alt_codes and alt_codes[lang1] in languages
```
Comment (JPJPJPOPOP) [inline trimpairs.py]: Too much indent here

**Review by JPJPJPOPOP — CHANGES_REQUESTED** [review]

File: apertium-languages.tsv (hunk 2 above)
Comment (dolphingarlic) [reply apertium-languages.tsv]: It's the coordinate of Hanover. I searched it in Google

File: trimpairs.py (hunk 1 above)
Comment (dolphingarlic) [reply trimpairs.py]: I used pylint. Should I change them back?

File: apertium-languages.tsv (hunk 2 above)
Comment (JPJPJPOPOP) [reply apertium-languages.tsv]: Hmm, can you change to 51,9? It's not a big change, but you should be getting the coordinate of the entire country (through Wikipedia), not some city in the country.

Comment (dolphingarlic) [reply apertium-languages.tsv]: Sure, I did that

Comment (dolphingarlic) [reply apertium-languages.tsv]: Should I add dut to it now?

File: trimpairs.py (hunk 3 above)
Comment (dolphingarlic) [reply trimpairs.py]: Fixed it

File: trimpairs.py (hunk 1 above)
Comment (jonorthwash) [reply trimpairs.py]: If there's a good reason to have changed them, that's fine.  I'm just curious what the reason was.

File: apertium-languages.tsv (hunk 2 above)
Comment (JPJPJPOPOP) [reply apertium-languages.tsv]: @dolphingarlic Since dut is the same as nld, we will just have two nodes that are on top of each other which isn't good. Anyways, I think your code is already converting the dut pairs to nld pairs which is fine.

Comment (JPJPJPOPOP) [reply apertium-languages.tsv]: I take that back, your code isn't converting dut pairs to nld pairs. There are no dut pairs at all in the pairs.txt.json file so there is no need to add it since nld already does the job.

**Review by JPJPJPOPOP — APPROVED** [review]
